src/actions/types.ts

```diff
@@ -27,32 +27,19 @@ export const SST_PAGE = 'SST_PAGE';
 export const SST_MENU = 'SUBMIT_MENU';
 export const SST_LOADING = 'SUBMIT_LOADING';
 
-export const FETCH_USER = 'FETCH_USER';
-
 export const MAIN_SCROLLER_BOTTOM = 'MAIN_SCROLLER_BOTTOM';
 
 export const NEW_EPOCH = 'NEW_EPOCH';
-export const WAIVER_CREATE = 'WAIVER_CREATE';
```

Comment:
What was reason to remove these actions? It is good to have in 1 place a list of all possible actions.

Reply:
These constants become useless in the Redux Toolkit approach. The action identifiers are automatically generated from the slice definition. For example, the `popAlert` action in the `waive` slice gets the name `waive/popAlert`.

Reply:
Please help me understand why only part of constants were dropped, while othe stay in the place?